src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -115,93 +149,97 @@ const CreateCourse = ({
-				...courseList,
-				{ ...newCourse, authors: courseAuthor.map((item) => item.id) },
-			]);
-			close(false);
+			newCourse = {
+				...newCourse,
+				duration: Number(duration),
+				authors: courseAuthor.map((item) => item.id),
+			};
+			fetchWithToken('courses/add', newCourse, token);
+			history.push('/courses');
 		}
 	};
 
 	return (
-		<form>
-			<StyledTop>
-				<div>
-					<div>Title</div>
-					<InputField
-						type='text'
-						name='title'
-						placeholder='Enter title'
-						onChange={handleTitle}
-					/>
-				</div>
-				<Button onClick={(e) => submitCourse(e, close)}>create course</Button>
-			</StyledTop>
-			<div>
-				<div>Description</div>
-				<DescriptionInput
-					type='text'
-					name='description'
-					placeholder='Enter description'
-					onChange={handleDescr}
-				/>
-			</div>
-			<StyledAuthors>
-				<StyledLeftColumn>
-					<div>Add author</div>
+		<CoursesContainer>
+			<form>
+				<StyledTop>
 					<div>
-						<div>Author's name</div>
-						<InputAuthor
+						<div>Title</div>
+						<InputField
 							type='text'
 							name='title'
-							placeholder='Enter author name'
-							ref={authorInput}
+							placeholder='Enter title'
+							onChange={handleTitle}
 						/>
 					</div>
-					<Button onClick={addAuthor}>create author</Button>
-				</StyledLeftColumn>
-				<StyledRightColumn>
-					<div>Authors</div>
-					{authorList.map((author) => {
-						return (
-							<StyledAuthorsList key={author.id}>
-								<div>{author.name}</div>
-								<Button onClick={(e) => addAuthorToList(e, author)}>
-									add Author
-								</Button>
-							</StyledAuthorsList>
-						);
-					})}
-				</StyledRightColumn>
-			</StyledAuthors>
-			<StyledAuthors>
-				<StyledLeftColumn>
-					<div>
-						<div>Duration</div>
-						<InputField
-							type='number'
-							name='duration'
-							placeholder='enter duration'
-							onChange={handleDuration}
-						/>
-					</div>
-					<h2>Duration: {timeConverter(duration)}</h2>
-				</StyledLeftColumn>
-				<StyledRightColumn>
-					<div>Course authors</div>
-					{courseAuthor.map((author) => {
-						return (
-							<StyledAuthorsList key={author.id}>
-								<div>{author.name}</div>
-								<Button onClick={() => removeAuthorToList(author)}>
-									delete Author
-								</Button>
-							</StyledAuthorsList>
-						);
-					})}
-				</StyledRightColumn>
-			</StyledAuthors>
-		</form>
+					<Button onClick={(e) => submitCourse(e, close)}>create course</Button>
+				</StyledTop>
+				<div>
+					<div>Description</div>
+					<DescriptionInput
+						type='text'
+						name='description'
+						placeholder='Enter description'
+						onChange={handleDescr}
+					/>
+				</div>
+				<StyledAuthors>
+					<StyledLeftColumn>
+						<div>Add author</div>
+						<div>
+							<div>Author's name</div>
+							<InputAuthor
+								type='text'
+								name='title'
+								placeholder='Enter author name'
+								ref={authorInput}
```

Comment:
Please, use state instead. Additionally, I'd recommend to clear author input after successful author creation